src/App.tsx

```diff
@@ -1,14 +1,46 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useCallback, useContext, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { getUser } from './api';
+import { TodosContext } from './components/TodosProvider';
 
 export const App: React.FC = () => {
+  const [userData, setUserData] = useState<User | null>(null);
+  const [currentTodo, setCurrentTodo] = useState<Todo | null>(null);
+  const [isModalLoading, setIsModalLoading] = useState(false);
+  const [loadUserDataError, setLoadUserDataError] = useState(false);
+
+  const { todos, loadTodosError } = useContext(TodosContext);
```

Comment:
😯

Reply:
Is it bad?

Reply:
no, awesome that you've practiced with creating the context)